guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequestService.java

```diff
@@ -0,0 +1,33 @@
+package org.guvnor.structure.repositories;
+
+import java.util.List;
+
+import org.uberfire.java.nio.base.FileDiff;
+
+public interface PullRequestService {
+
+    PullRequest createPullRequest( PullRequest pullRequest );
+
+    PullRequest acceptPullRequest( String repository,
+                                   long id );
+
+    PullRequest rejectPullRequest( String repository,
+                                   long id );
+
+    PullRequest closePullRequest( String repository,
+                                  long id );
+
+    List<PullRequest> getAllPullRequestsFor( String repository,
```

Comment:
We should try to follow some of the standards for filtering, pagination and ordering in our services. I would recommend to take a look at this, to follow the same approach: 
- Paging and Sorting -> https://github.com/droolsjbpm/guvnor/blob/master/guvnor-ala/guvnor-ala-spi/src/main/java/org/guvnor/ala/registry/RuntimeRegistry.java#L44
- For filtering I like more descriptive names.. like for example getPullRequestsByRepository( String repositoryName) , we should get rid of the "All" and add the Paging and Sorting parameters